static/sass/_pattern_global-nav.scss

```diff
@@ -0,0 +1,105 @@
+@mixin canonical-p-global-nav {
+
+  // Using id to override inherited styles from Vanilla and project
+  #g-navigation {
+    height: 2rem;
+    line-height: 1rem;
+    background-color: $color-dark;
```

Comment:
pleasee fix linting issues, the pipeline is failing

Reply:
I'll do it once UX +1 otherwise the code will keep changing and will need to re-order alphabetically again.